components/mpas-seaice/bld/namelist_files/namelist_definition_mpassi.xml

```diff
@@ -320,14 +320,6 @@ Valid values: Any positive integer.
 Default: Defined in namelist_defaults.xml
 </entry>
 
-<entry id="config_nFloeCategories" type="integer"
-	category="dimensions" group="dimensions">
-The number of ice floe categories to use.
-
-Valid values: 1(default), 12, 16, 24.
```

Comment:
I think the specific values will be the only ones allowed when coupled with WW3, and in CICE we only test 1 and 12 categories.  But "any positive integer" will do for now.

Reply:
I'll make that change once I find out what's going on with ice-ocean drag, as per my comment below.